src/main/steps/common/webpack/js.njk

```diff
@@ -1 +1 @@
- <script src="/main-dev.js"></script>
```

Comment:
what is the change here @hmcts-abduldaim

Reply:
@yogendraU  this is already in the base Branch. it's came as a rebased. it is required